Logging/.CSProject/Log.cs

```diff
@@ -142,18 +142,18 @@ private static bool ShouldIgnoreAssembly(Assembly assembly)
         /// <summary>
         /// Creates a <see cref="Logger"/> instance for a given instance.
         /// </summary>
-        /// <param name="instance">The instance to create the <see cref="Logger"/> instance for.</param>
+        /// <param name="owner">The object to create a <see cref="Logger"/> instance for.</param>
```

Comment:
It just seemed more descriptive to refer to the given object as the Logger's "owner" rather than simply an "instance" (I mean, everything that isn't a primitive is an object instance, right?) I can change it back to `instance` though, just trying some things to see what fit best.

Reply:
oh `owner` is fine as a param. That or `context`. 
I was more referring to the use of instance in the comment.

If instance is everything non-primitive then object is everything including primitive!